client/components/chart/bar.jsx

```diff
@@ -29,34 +28,30 @@ module.exports = React.createClass( {
 	},
 
 	buildSections: function() {
-		var value = this.props.data.value,
-			max = this.props.max,
-			percentage = max ? Math.ceil( ( value / max ) * 10000 ) / 100 : 0,
+		const { active, data, max } = this.props;
+		const { nestedValue, value } = data;
+
+		const percentage = max ? Math.ceil( ( value / max ) * 10000 ) / 100 : 0,
```

Comment:
This expression looks a bit confusing to me the way it depends on `max`. Maybe this would be a good opportunity. No worries if not though.

```js
static propTypes = {
	max: PropTypes.number
}

static defaultProps = {
	max: Infinity
}

// If `max` isn't provided, it will be Infinity and `value / max`
// will collapse down to zero, giving the same result as before
const percentage = Math.ceil( ( value / max ) * 10000 ) / 100;
```